### ISSUE TITLE
[Bug] Wan2.1-14B-I2V VSA acceleration is not satisfactory

### ISSUE THREAD

**Issue by qinziqiao** [hao-ai-lab/FastVideo#868]
### Describe the bug

I attempted to use VSA in Wan2.1-14B-I2V-720P with a high sparsification rate of 0.95. However, during the inference process, H20 inference only accelerated by 3 times, while H100 inference accelerated by 2 times. Is this normal?

### Reproduction

models: Wan2.1-14B-I2V-720P

### Environment

pytorch==2.8.0
vsa==0.03

**Comment by Lyxien** [comment]
Which script are you using? As far as I remember, training DMD + VSA on Wan2.1-14B-I2V-720P isn’t supported at the moment.

**Comment by qinziqiao** [comment]
@Lyxien  hi, I did not use DMD training. Instead, we replaced Wan finetuning with VSA for speed measurement.

**Comment by EricLina** [comment]
1. I remember that the CUDA version of VSA doesn't support H20.  
2. You may be using the Triton version of VSA, which is slightly slower than FlashAttention for small token numbers.

**Comment by EricLina** [comment]
I think it is normal because GPU types have a significant influence on the attention kernel acceleration.

**Comment by Lyxien** [comment]
@EricLina In my view, the speedup isn’t very noticeable, mainly because padding significantly increases the token count.

**Comment by EricLina** [comment]
@Lyxien  Many sparse attention algorithms/kernels only speed up by 1~3 times, which is related to some inconsistent IO caused by the dynamic sparse pattern. Meanwhile, the CUDA backend would be significantly faster than the Triton backend; however, VSA(CUDA) currently only supports `arch>=sm_90a`.

**Comment by Lyxien** [comment]
@EricLina I see. If this is a Triton-related issue, it could very well be one of the primary causes of the performance degradation. Moreover, many sparse attention approaches take I/O affinity into consideration (e.g., aligning sparse block sizes with FlashAttention tile sizes). Was this aspect considered in VSA?

**Comment by EricLina** [comment]
> [@EricLina](https://github.com/EricLina) I see. If this is a Triton-related issue, it could very well be one of the primary causes of the performance degradation. Moreover, many sparse attention approaches take I/O affinity into consideration (e.g., aligning sparse block sizes with FlashAttention tile sizes). Was this aspect considered in VSA?

Not yet.
